### ISSUE TITLE
Filter links

### ISSUE THREAD

**Issue by michidk** [mafisz/pagekit-sitemap#2]
Hey,

would it possible to filter / exclude some links from the sitemap?
Or is there a way to get all navigation entrys from pagekit, so that the plugin could add the pages to the sitemap directly?

Because now, there a lot of unwanted links in my sitemap:
`http://voxvr.brokenvector.net/sitemap.xml`
like that one: http://myDomain.net/cdn-cgi/l/email-protection#1b7874756f7a786f5b796974707e756d7e786f746935757e6f (which is the mail protection from cloudflare). And the blog entries are always existing twice: the normal link and that one with #comments: http://myDomain.net/blog/4#comments

**Comment by mafisz** [comment]
Yes I think about that, and probably some filtering will be added, but I can't get all routes directly right now.

**Comment by michidk** [comment]
Note: the linked sitemap doesn't show the wrong links anymore, because i removed the manually.

Does your plugin update the sitemap.xml after a page gets added or do i have to regenerate them manually?

**Comment by mafisz** [comment]
No, it should be generated manualy. 
Again I think about that :) but i'm still learning Pagekit. It can take some time but maybe I found the solution.
